assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -222,22 +222,20 @@ function recurringPaymentAuthorisationHandler(
   )));
 }
 
-const recurringPaymentAuthorisationHandlers: {
-  [PaymentMethod]: (Dispatch<Action>, State, PaymentAuthorisation) => void
-} = {
+// Bizarrely, adding a type to this object means the type-checking on the
+// paymentAuthorisationHandlers is no longer accurate.
+// (Flow thinks it's OK when it's missing required properties).
+const recurringPaymentAuthorisationHandlers = {
   // These are all the same because there's a single endpoint in
   // support-frontend which handles all requests to create a recurring payment
   PayPal: recurringPaymentAuthorisationHandler,
   Stripe: recurringPaymentAuthorisationHandler,
   DirectDebit: recurringPaymentAuthorisationHandler,
 };
 
-const paymentAuthorisationHandlers: {
-  [Contrib]: {
-    [PaymentMethod]: (Dispatch<Action>, State, PaymentAuthorisation) => void
-  }
-} = {
+const paymentAuthorisationHandlers: PaymentMatrix<(Dispatch<Action>, State, PaymentAuthorisation) => void> = {
   ONE_OFF: {
+    ...baseHandlers.ONE_OFF,
```

Comment:
Like you, I'm in 2 minds as to whether the `baseHandlers` makes things better here. I'll leave it up to you

Reply:
I'll leave it in for now.